docs/source/_static/css/huggingface.css

```diff
@@ -125,6 +125,12 @@ a.copybtn {
     background-color: #6670FF;
 }
 
+/* The section headers in the toc tree */
+.wy-menu-vertical p.caption{
+    background-color: #4d59ff;
+    line-height: 40px;
```

Comment:
This is so that the title appears centered in the darker strip, with the value inherited (32px) it was too much on the bottom.